ts/client/src/client.ts

```diff
@@ -321,7 +322,10 @@ export class MangoClient {
 
   public async getGroup(groupPk: PublicKey): Promise<Group> {
     const groupAccount = await this.program.account.group.fetch(groupPk);
-    const group = Group.from(groupPk, groupAccount);
+    const group = Group.from(
+      groupPk,
+      groupAccount as IdlAccounts<MangoV4>['group'],
```

Comment:
I thought `this.program.account.group.fetch()` and fiends already gave us the right type, so would think that this cast isn't needed.

After all Group.from() demands a specific type for the second argument and there should have been type checking errors if they were incompatible.